### ISSUE TITLE
bad formatting for inline comment for return value

### ISSUE THREAD

**Issue by andreimatei** [cockroachdb/crlfmt#29]
I want this:

```
func (r *limitedCommandResult) AddRow(
  ctx context.Context, row tree.Datums,
) (_ /* cont */ bool, _ error) {
```

the tool gives me this:
```
func (r *limitedCommandResult) AddRow(
	ctx context.Context, row tree.Datums,
) (
	_ bool, /* cont */
	_ error,
) {
```

**Comment by benesch** [comment]
Yes, that's by design. If you want to document a result parameter with an inline comment then it deserves its own line IMO. Also, in this case I'd just name the result parameter:

```
func (r *limitedCommandResult) AddRow(
  ctx context.Context, row tree.Datums,
) (cont bool, _ error) {
```

Since you're the biggest proponent of comments in result params I'm happy to defer to your style preferences. I won't have time to fix this, though, so punting the assignment back to you. The code you'd have to change is around here: https://github.com/cockroachdb/crlfmt/blob/3c56c65c97a11c610354ce15b9939c0d0aff5d7a/internal/render/render.go#L190-L197
